ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -43,17 +43,22 @@
         <AlertInline @type="danger" @message={{join ", " this.modelValidations.name.errors}} />
       {{/if}}
       <SearchSelect
-        @subLabel="Entities"
+        @id="entities"
+        @label="Entities"
+        @labelClass="is-label"
         @placeholder="Search"
         @models={{array "identity/entity"}}
+        @inputValue={{map-by "id" @model.entity_ids}}
```

Comment:
Claire - the inputValue is what is used on searchSelect to make the this.selectedOptions, which is what shows the list of already selected ids (in this case entities, in yours assignments). If I log above this `{{log (map-by "id" @model.entity_ids)}} it shows an array like yours does, but for some reason that's not coming through on my SearchSelect like it is yours.

Reply:
We need to wait until the promise for the `entity_ids` and `group_ids` is fulfilled, so this search select needs to be wrapped in a conditional. I'm pushing a commit to fix this!